packages/host/app/components/operator-mode/card-inheritance-panel.gts

```diff
@@ -10,88 +10,118 @@ import {
   @tracked cardInstance: CardDef | undefined;
   @service declare operatorModeStateService: OperatorModeStateService;
 
+  get cardType() {
+    return this.args.adoptionChainManager?.selectedElement?.cardType;
+  }
+
+  get elementsInFile() {
+    return this.args.adoptionChainManager?.elementsInFile;
+  }
+
+  get isLoading() {
+    return (
+      this.args.adoptionChainManager?.isLoading || this.cardType?.isLoading
+    );
+  }
+
   @action
   updateCodePath(url: URL | undefined) {
     if (url) {
       this.operatorModeStateService.updateCodePath(url);
     }
   }
 
-  get adoptionChainTypes() {
-    return this.args.adoptionChain?.types;
+  @action
+  select(el: ElementInFile) {
+    this.args.adoptionChainManager?.select(el);
   }
 
   <template>
     <div class='container' ...attributes>
-
-      {{#if @cardInstance}}
-        {{! JSON case when visting, eg Author/1.json }}
-        {{#each this.adoptionChainTypes as |t|}}
-          <InstanceDefinitionContainer
-            @name={{@cardInstance.title}}
-            @fileExtension='.JSON'
-            @realmInfo={{@realmInfo}}
-            @realmIconURL={{@realmIconURL}}
-            @infoText={{this.lastModified}}
-            @actions={{array
-              (hash label='Delete' handler=@delete icon='icon-trash')
-            }}
-          />
-          <div>Adopts from</div>
-
-          <ClickableModuleDefinitionContainer
-            @name={{t.displayName}}
-            @fileExtension={{this.fileExtension}}
-            @realmInfo={{@realmInfo}}
-            @realmIconURL={{@realmIconURL}}
-            @onSelectDefinition={{this.updateCodePath}}
-            @url={{t.module}}
-          />
-        {{/each}}
+      {{#if this.isLoading}}
+        <div>Loading...</div>
       {{else}}
-        {{! Module case when visting, eg author.gts }}
-        {{#each this.adoptionChainTypes as |t|}}
+        {{#if @cardInstance}}
+          {{! JSON case when visting, eg Author/1.json }}
+          <h3>Inheritance Panel</h3>
+          <div class='inheritance-chain'>
+            <InstanceDefinitionContainer
+              @name={{@cardInstance.title}}
+              @fileExtension='.JSON'
+              @realmInfo={{@realmInfo}}
+              @infoText={{this.lastModified}}
+              @actions={{array
+                (hash label='Delete' handler=@delete icon='icon-trash')
+              }}
+            />
+            <div>Adopts from</div>
+
+            <ClickableModuleDefinitionContainer
+              @name={{this.cardType.type.displayName}}
+              @fileExtension={{this.cardType.type.moduleMeta.extension}}
+              @realmInfo={{this.cardType.type.moduleMeta.realmInfo}}
+              @onSelectDefinition={{this.updateCodePath}}
+              @url={{this.cardType.type.module}}
+            />
+          </div>
+        {{else}}
+          {{! Module case when visting, eg author.gts }}
+          <h3>In This File</h3>
+          {{#each this.elementsInFile as |el|}}
```

Comment:
> Would it be more appropriate to name this `fileExports`? Yesterday in pairing we were referring to these as "exports".

oh you are referring to the name of the variable?? Hmmm I used `elementsInFile` because im referring to just something that will be displayed in the "In This File" panel

Reply:
Yes, referring to the variable. Maybe "elements" is not descriptive enough of what this collection holds. That's why i'm suggesting "exports" since we were using that name when we paired

Reply:
What is the difference between: elements, exports, definitions?

Reply:
- Elements: card | field | class | function (exported or unexported)
- Definition: card | field (exported or unexported)

At least for now. I can see how the naming of the UI definition-container may mislead when we are considering non-defintions and I somehow or another use the same components. 

The future work is to make the module resource include unexported fields